plugins/devo/plugin.spec.yaml

```
@@ -0,0 +1,270 @@
+      description: "Client IP address"
+      required: false
+    cookie:
+      title: "Cookie"
+      type: string
+      description: "Cookie"
+      required: false
+    eventdate:
+      title: "Event Date"
+      type: integer
+      description: "Event date"
+      required: false
+    method:
+      title: "Method"
+      type: string
+      description: "Method"
+      required: false
+    protocol:
+      title: "Protocol"
+      type: string
+      description: "Protocol"
+      required: false
+    referralUri:
+      title: "Referral URI"
+      type: string
+      description: "Referral URI"
+      required: false
+    statusCode:
+      title: "Status Code"
+      type: integer
+      description: "Status code"
+      required: false
+    timeTaken:
+      title: "Time Taken"
+      type: integer
+      description: "Time taken"
+      required: false
+    timestamp:
+      title: "Timestamp"
+      type: string
+      description: "Timestamp"
+      required: false
+    uri:
+      title: "URI"
+      type: string
+      description: "URI"
+      required: false
+    userAgent:
+      title: "User Agent"
+      type: string
+      description: "User agent"
+      required: false
+  query_result:
+    msg:
+        title: "Message"
+        type: string
+        description: "Message"
+        required: false
+    timestamp:
+        title: "Timestamp"
+        type: integer
+        description: "Timestamp"
+        required: false
+    cid:
+        title: "CID"
+        type: string
+        description: "CID"
+        required: false
+    status:
+        title: "Status"
+        type: integer
+        description: "Status"
+        required: false
+    object:
+        title: "Log Entries"
+        type: "[]log_entry"
+        description: "Log entries"
+        required: false
+
+connection:
+  authentication_token:
+    title: Authentication Token
+    description: Devo authentication token
+    type: credential_secret_key
+    required: true
+    example: 9de5069c5afe602b2ea0a04b66beb2c0
+  region:
+    title: Region
+    description: Region
+    type: string
+    required: true
+    example: USA
+    enum:
+      - USA
+      - EU
+      - VDC (Spain)
+
+triggers:
+  get_new_alerts:
+    title: Get New Alerts
+    description: Get new alerts
+    output:
+      alert:
+        title: Alert
+        description: An alert
+        type: object
+        required: true
+
+actions:
+  query_logs:
+    title: Query Logs
+    description: Run a query against the logs
+    input:
+      query:
+        title: Query
+        description: A Query. The response be limited to 200 mb of raw data or 1000 entries whichever is hit first
+        type: string
+        required: true
+        example: "from demo.ecommerce.data select *"
+      from_date:
+        title: From Date
+        description: Earliest date to query events from, will accept relative or absolute times, e.g. 1/1/2020, 2 hours ago, 1/1/2020T12:00:00, Now
```

```suggestion
        description: Earliest date to query events from, will accept relative or absolute times, e.g. '1/1/2020', '2 hours ago', '1/1/2020T12:00:00', or 'Now'
```
Single quotes can help make this more readable. Also - a thought I just had - the relative timestamps are not relative to the time of workflow creation - it's relative to the time the Python code actually executes. This should be made clear to the user, otherwise we'll likely experience user confusion.